### ISSUE TITLE
Npm run test fail (W10)

### ISSUE THREAD

**Issue by NicolasBouhours** [Hashnode/mern-starter#287]
Hi

I've generate project with mern-cli, and i got an issue with "npm run test" command, all other works perfectly. I use windows 10 64 bits.

This is the error : 

```
C:\Project\React\Mern\mern-starter\node_modules\.bin\ava:2
basedir=$(dirname "$(echo "$0" | sed -e 's,\\,/,g')")
          ^^^^^^^
SyntaxError: missing ) after argument list
```

If you have any suggestion for fixing it !
Thank's :)

**Comment by b1ackmartian** [comment]
+1

**Comment by mannyhenri** [comment]
@MrChristofferson up to take a look at the test issues? This issue might be related to #250

**Comment by mannyhenri** [comment]
Fixed with PR #370
